.github/workflows/publish-npm.yml

```diff
@@ -61,3 +61,12 @@ jobs:
             }
         env:
           SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
+      - name: Send Repository Dispatch event to bpmn-visualization-R repository
+        uses: peter-evans/repository-dispatch@v1
+        with:
+          token: ${{ secrets.GH_RELEASE_TOKEN }}
+          repository: process-analytics/bpmn-visualization-R
+          event-type: update_bpmn_visualization_version
+          client-payload: '{
+              "version": "${{ steps.get_version.outputs.VERSION }}"
```

Comment:
⚠️ the `get_version` step doesn't exist, so the version won't be set here!

Reply:
Good catch 👍🏻